node/resolvers/submitStartupForm.ts

```diff
@@ -29,13 +29,15 @@ export async function submitStartupForm(
     await service.getConvertedExternalData()
 
   const {
-    body: { storeName },
+    body: { storeName, affiliateId, email },
   } = args
 
   const appSettings = appSettingsService(apps, log)
 
-  await appSettings.saveValueOnAppSettings('storeName', storeName)
-  await appSettings.saveValueOnAppSettings(
+  await appSettings.saveValueToAppSettings('storeName', storeName)
+  await appSettings.saveValueToAppSettings('email', email)
+  await appSettings.saveValueToAppSettings('affiliateId', affiliateId)
+  await appSettings.saveValueToAppSettings(
```

Comment:
```suggestion
  await appSettings.saveValueOnAppSettings('storeName', storeName)
  await appSettings.saveValueOnAppSettings('email', email)
  await appSettings.saveValueOnAppSettings('affiliateId', affiliateId)
  await appSettings.saveValueOnAppSettings(
```

Build is failing because of this. I refactored the name of this method in the last PR.